internal/cli/cli.go

```diff
@@ -83,6 +87,24 @@ func (opts *deleteOpts) DeleteFromProject(d DeleterFromProject, projectID string
 	return nil
 }
 
+// DeleterFromProjectAuthDB deletes a resource from a project, it expects a callback
```

Comment:
I do not why after this change every time I run a test regarding any delete I get `=== RUN   TestAtlasDBUsersDelete_Run
%!(EXTRA string=test)--- PASS: TestAtlasDBUsersDelete_Run (0.00s)
PASS`. 

There is this extra string **(EXTRA string=test)**, and I do not know where it came from. I also tried to revert all of my changes but I still get the same result 🤕

Reply:
I fixed the issue by adding the successMessage to the test.  The problem was caused by `fmt.Printf(opts.successMessage, opts.entry)` inside the function **DeleterFromProject** which was executed without the successMessage

Reply:
can we improve on the `deleteOpts` api then? let's do this in another PR but this was probably an overlook from my side, we could improve this by having a fallback message and check the value we are using, like a said separate PR to not complicate this too much